NodePhantomJS/entry_point.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+export GEOMETRY="$SCREEN_WIDTH""x""$SCREEN_HEIGHT""x""$SCREEN_DEPTH"
+
+if [ -z "$HUB_PORT_4444_TCP_ADDR" ]; then
+  echo Not linked with a running Hub container 1>&2
+  exit 1
+fi
+
+function shutdown {
+  kill -s SIGTERM $NODE_PID
+  wait $NODE_PID
+}
+
+ip="$(ifconfig | grep -A 1 'eth0' | tail -1 | cut -d ':' -f 2 | cut -d ' ' -f 1)"
+
+phantomjs --webdriver=$ip:8080 --webdriver-selenium-grid-hub=http://$HUB_PORT_4444_TCP_ADDR:$HUB_PORT_4444_TCP_PORT
```

Comment:
Phantomjs (as far as I know) still is not fully integrated into WebDriver.  It is, however, integrated with Ghostdriver, which allows it to register itself as a node to a selenium grid hub.

You could try telling webdriver to launch phantomjs with a config, but it wouldn't work.

https://groups.google.com/forum/#!topic/webdriver/HA5C9Jnpr7g